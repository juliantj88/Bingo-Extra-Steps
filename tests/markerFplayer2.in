1
test1
test2
1 R 5
2 U 1
5 R 5
4 U 5
3 Y 4
0 B 3
0 Y 4
0 L 4
0 U 1
save markerFplayer2
4