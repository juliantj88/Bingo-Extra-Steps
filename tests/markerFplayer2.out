test1
test2
2
1
0
F...............
RRBB
RRYY
RLLY
BBUU
RBYY
byrUl
lbyru
ulbyr
rulbY
yRulb
byrUl
lbyru
ulByr
rulby
yrulb
.
..
...
....
.....
.
..
...
LL..
UUU..
       
       

RRLBBYRLLUYBURRBLRYRBLBUUBRLBLULYRLYLLUYYULLLRLUYBLBYYYRBUURUBBYUBYRU
0